Backend/BlockChangesService.cs

```diff
@@ -63,7 +68,7 @@ private async Task SetDoneStatus(BlockChangeRequest request)
         {
             if (request.Status == ExchangeStatus.Done)
             {
-                throw new ArgumentException("You cannot exchange requests which are already exchanged!");
+                await new Task(() => { throw new ArgumentException("You cannot exchange requests which are already exchanged!"); });
```

Comment:
Why do you have to throw like this? Isn't simple throw enough?

Reply:
it is because of warning sonarlint...